testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adduser/AddUserTest.java

```diff
@@ -0,0 +1,106 @@
+package org.keycloak.testsuite.adduser;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.jboss.arquillian.container.test.api.ContainerController;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.keycloak.admin.client.resource.RealmResource;
+import org.keycloak.admin.client.resource.UserResource;
+import org.keycloak.credential.hash.Pbkdf2Sha256PasswordHashProviderFactory;
+import org.keycloak.models.Constants;
+import org.keycloak.representations.idm.*;
+import org.keycloak.testsuite.AbstractKeycloakTest;
+import org.keycloak.util.JsonSerialization;
+import org.keycloak.wildfly.adduser.AddUser;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author <a href="mailto:mabartos@redhat.com">Martin Bartos</a>
+ * @author <a href="mailto:sthorger@redhat.com">Stian Thorgersen</a>
+ */
+public class AddUserTest extends AbstractKeycloakTest {
+
+    @ArquillianResource
+    private ContainerController controller;
+
+    @Test
+    public void addUserTest() throws Exception {
+        final String username = "addusertest-admin";
+        final String configDir = System.getProperty("auth.server.config.dir");
```

Comment:
Are you sure "auth.server.config.dir" is always defined? You can use `getProperty(String key, String def)` instead or you can at least check the configDir is not null afterwards.